kraken_biom.py

```diff
@@ -6,6 +6,7 @@
 """
 from __future__ import absolute_import, division, print_function
 
+from pathlib import Path
```

Comment:
This will require dropping support for all python versions <3.4, which I'm fine with, so I'm just mentioning this for future reference since this could technically be implemented with the basic glob module if needed.